e2e/app/eoa/static.go

```diff
@@ -10,6 +10,7 @@ var statics = map[netconf.ID][]Account{
 		wellKnown(DevPrivateKey0(), RoleCreate3Deployer, RoleDeployer, RoleManager, RoleUpgrader),
 		wellKnown(DevPrivateKey5(), RoleRelayer),
 		wellKnown(DevPrivateKey6(), RoleMonitor),
+		wellKnown(DevPrivateKey2(), RoleFlowgen),
```

Comment:
you can reorder devaccounts to be in increasing order, it doesn't have to remain consistently mapped.